features/providers/means_report.feature

```diff
@@ -168,6 +169,174 @@ Feature: Means report
       | question |
       | Uploaded bank statements |
 
+  Scenario: For a non-passported enhanced bank statement upload journey
```

Comment:
How do you feel about moving this scenario to it's own file in `features/providers/enhanced_bank_upload/means_report.feature`?

I think it helps the maintainability of feature tests if they are broken up into more manageable pieces.

I'm pretty sure @colinbruce has done something similar with some of the mini-loop proceeding work, but we haven't taken time to apply this approach consistently across all tests yet.

Reply:
There is probably a wider question or organising features along the three fundamental journey. 
i think @colinbruce  done that to keep the loop work isolated in the short/medium term.

Eventually the enhanced journey will be the only bank statement upload journey at which point i would hope merge them into existing feature tests organised by "feature" - I rather like working on all means report in the one feature file and using a scenarios for the "journey".